Dockerfile.template

```diff
@@ -12,7 +12,7 @@ RUN set -ex; \
     ; \
     rm -rf /var/lib/apt/lists/*; \
     \
-    docker-php-ext-install mbstring opcache pdo pdo_mysql
+    docker-php-ext-install mbstring opcache pdo pdo_mysql mysqli
```

Comment:
Hmm, I would prefer to keep pdo only, but OK, may be easier to start from scratch with mysqli.